app/src/main/java/unipd/se18/ocrcamera/PhotoDownloadTask.java

```diff
@@ -0,0 +1,216 @@
+package unipd.se18.ocrcamera;
+
+import android.app.Activity;
+import android.content.Context;
+import android.os.AsyncTask;
+import android.os.Environment;
+import android.util.Log;
+import android.widget.Button;
+import android.widget.ProgressBar;
+import android.widget.ScrollView;
+import android.widget.TextView;
+
+import org.apache.commons.net.ftp.FTPClient;
+import org.apache.commons.net.ftp.FTPFile;
+import org.apache.commons.net.ftp.FTPReply;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+
+
+/**
+ * @author Leonardo Rossi (g2) and Stefano Romanello (g3)
+ */
+public class PhotoDownloadTask extends AsyncTask<Void, Integer, Void>
+{
+
+    private FTPClient ftp;
+    //private ArrayList<String> messages;
+    private Context context;
+    private ProgressBar progressBar;
+    private TextView textViewProgress;
+    private Button button;
+    private Integer currentProgress;
+    private ScrollView scrollCurrentDownload;
+    private TextView txtViewCurrentDownload;
+    //Constants
+    // PLEASE DON'T USE THESE FOR PERSONAL USAGE!
+    // FTP CREDENTIALS
+    // USERNAME: epiz_22864730
+    // PASSWORD: 8M0tNtsJCsw
+    // HOSTNAME: ftpupload.net
+    // PLEASE DON'T USE THESE FOR PERSONAL USAGE!
+    // Communicate with Group 3 if you need something from the server.
+    private final String USERNAME = "epiz_22864730";
+    private final String PASSWORD = "8M0tNtsJCsw";
+    private final String HOSTNAME = "ftpupload.net";
+    private final String PHOTOS_FOLDER = Environment.getExternalStoragePublicDirectory(Environment.DIRECTORY_PICTURES)+"/OCRCameraDB";
```

Comment:
Could this variable be protected and static for using in the testing area?

Reply:
for now let them as they are please, I dont want the credential to be everywhere. I'm worried that someone use them in a bad way with ftpclient and risk to delete everything.

Reply:
PHOTOS_FOLDER declared in the same way is already used in other testing classes, no need to set this static

Reply:
Exactly, could be a good idea writing this variable once and sharing it to the other class that uses it?